resources/tenant-onboarding.yaml

```diff
@@ -1417,6 +1556,29 @@ Resources:
         ECSSecurityGroup: !Ref ECSSecurityGroup
         EncryptEFS: !Ref EncryptEFS
         EFSLifecyclePolicy: !Ref EFSLifecyclePolicy
+  # ElasticSearch Extension
+  es:
+    Type: AWS::CloudFormation::Stack
+    Condition: ProvisionES
+    Properties:
+      TemplateURL: !Sub https://${SaaSBoostBucket}.s3.amazonaws.com/tenant-onboarding-es.yaml
+      TimeoutInMinutes: 45
```

Comment:
45 minutes for timeout?! is this maybe accounting for the fact that we could relaunch a tenant with existing data that would take a long time to load? i noticed in the referenced PR that this was 30, which is still a fairly large value